### PR TITLE
JSTableToMarkdown array values

### PR DESCRIPTION
**Pull request by liorblob**
master currently fails

### CODE REVIEW

File: TestPlaybooks/script-VerifyTableToMarkDown.yml [hunk 1]
```diff
@@ -32,6 +36,16 @@ script: |-
       throw "Simple Data failed to be converted to RIGHT Human Redable fotmat. Got: " + h;
   }
 
+  h = tableToMarkdown("Simple Array", JSON.parse(simpleArray), 'Array');
```
Comment (anara123) [inline TestPlaybooks/script-VerifyTableToMarkDown.yml]: why you define simpleArray as a string and then `JSON.parse` it? Why not define it as array from the beginning

File: TestPlaybooks/script-VerifyTreeToFlatObject.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
+commonfields:
+  id: c5cb179f-d6d2-4d87-8857-b224689d5b00
+  version: 15
+name: VerifyTreeToFlatObject
+script: |+
+  var dataWithNestedObject = '{'+
```
Comment (anara123) [inline TestPlaybooks/script-VerifyTreeToFlatObject.yml]: same here with, why not just define the object

File: TestPlaybooks/script-VerifyTableToMarkDown.yml (hunk 1 above)
Comment (liorblob) [reply TestPlaybooks/script-VerifyTableToMarkDown.yml]: I was being consistent with the `VerifyTableToMarkdown` script, I assume there was a reason for it

Comment (anara123) [reply TestPlaybooks/script-VerifyTableToMarkDown.yml]: I don't see a valid reason reading the script. Please change this in all the places in the script to be object

**Pull request merged**